dtk/process.py

```diff
@@ -438,17 +438,29 @@ def curve_area_stats(x, y):
     return xstats
 
 
-def freq_spectrum(data, sampleRate):
+def freq_spectrum(data, sample_rate, norm="forward", remove_dc_component=True):
```

Comment:
```suggestion
def freq_spectrum(data, sampleRate, norm="forward", remove_dc_component=True):
```

We technically shouldn't change the one that is already there because you can call the function as:

```python
freq_spectrum(data=data, sampleRate=sample_rate)
```

but new things we add should use pep8 style for variable names.